src/connector/src/source/cdc/external/mod.rs

```diff
@@ -362,7 +365,7 @@ impl ExternalTableImpl {
                 .await?,
             )),
             CdcSourceType::SqlServer => Ok(ExternalTableImpl::SqlServer(
-                SqlServerExternalTable::connect(config).await?,
+                SqlServerExternalTable::connect(config, defined_column_descs).await?,
```

Comment:
How does other CDC tables (PostgreSQL / MySQL) verify user specified column data types against derived ones? In Java part?

Reply:
Yes, actually those cdc(pg, mysql, mssql) all implement the schema check logic in` xxxValidater.java`, but somehow the sqlserver cdc bypass this check and I just found we need to do double check here.

Reply:
> somehow the sqlserver cdc bypass this check and I just found we need to do double check here

Could you elaborate on `somehow`? Is such double checking already neccessary for pg / mysql?